### PR TITLE
Mtm50953 improvements for enterprise tenant documentation

### PR DESCRIPTION
**Pull request by dianaolives**

### CODE REVIEW

File: content/users-guide/enterprise-tenant-bundle/managing-tenants.md [hunk 1]
```diff
@@ -39,17 +39,20 @@ To be able to use the tenant functionality, your user needs to have the appropri
...
 * The name of the subtenant, for example, company name of your customer.
 * The ID and domain.
-* Optionally, a contact name and phone number.
+* Optionally, a contact name.
 * The date when the tenant was created.
 * The status of the tenant, either active (indicated by a green checkmark icon) or suspended (indicated by a red cross icon).
 
-In the {{< management-tenant >}}, you will also find information on the parent tenant, that is, the tenant that created the listed tenant.
+{{< c8y-admon-info >}}
+In the {{< management-tenant >}}, you will also find a column with information on the parent tenant. The parent tenant is the original tenant that created the subtenants that are listed on the table.
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/managing-tenants.md:53]:
```suggestion
In the {{< management-tenant >}}, you will also find a column with information on the parent tenant. The parent tenant is the original tenant that created the subtenants that are listed in the table.
```

File: content/users-guide/enterprise-tenant-bundle/managing-tenants.md [hunk 2]
```diff
@@ -177,12 +180,11 @@ If data broker connectors are configured for a tenant, suspending this tenant re
...
 	![Suspend tenant](/images/users-guide/enterprise-tenant/et-subtenant-suspend.png)
 
-2. In the resulting dialog box confirm the suspension by clicking **OK** and entering your password.
-
-As part of suspending the tenant, an email is sent to the tenant administrator if an email address is configured for that administrator.
+2. In the resulting dialog box confirm the suspension by clicking the confirmation button and then entering your password.
 
 {{< c8y-admon-info >}}
-If you are a service provider, you can suppress this email.
+* As part of suspending the tenant, an email is sent to the tenant administrator to the email adress that was configured for that administrator.
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/managing-tenants.md:186]:
```suggestion
* As part of suspending the tenant, an email is sent to the the email adress that was configured for the tenant administrator.
```

File: content/users-guide/enterprise-tenant-bundle/support-user-access.md [hunk 3]
```diff
@@ -47,7 +47,7 @@ Details on the status of support requests and support user access for a tenant c
 
 To log in as support user from the {{< management-tenant >}}, you must provide the following information in the Login screen:
 
-* Tenant ID - the ID of the tenant of the user to be supported. The tenant ID is shown in the user dropdown menu in the UI. In some environments, especially in test environments, you can open the destination tenant´s URL (for example *testtenant.{{< domain-c8y >}}*) and log into the tenant without the tenant ID.
+* Tenant ID - the ID of the tenant of the user to be supported. The tenant ID is shown in the user dropdown menu in the UI.
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/support-user-access.md:50]:
```suggestion
* Tenant ID - the ID of the tenant of the user to be supported. The tenant ID is shown in the user menu in the UI.
```

File: content/users-guide/enterprise-tenant-bundle/managing-tenants.md (hunk 2 above)
Comment (dianaolives) [reply content/users-guide/enterprise-tenant-bundle/managing-tenants.md:186]: ok

File: content/users-guide/enterprise-tenant-bundle/managing-tenants.md (hunk 1 above)
Comment (dianaolives) [reply content/users-guide/enterprise-tenant-bundle/managing-tenants.md:53]: ok

File: content/users-guide/enterprise-tenant-bundle/managing-tenants.md [hunk 4]
```diff
@@ -347,7 +349,7 @@ In the **Default subscriptions** page, you can configure two separate lists of a
 - Every existing tenant on platform upgrade.
 
 {{< c8y-admon-info >}}
-These default lists can be overridden for particular subtenants by setting additional tenant options, for example via tenant policy. For details, see [Default subscriptions](#default-subscriptions) below or the [Tenant API](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#tag/Tenant-API) in the {{< openapi >}}.
+These default lists can be overridden for particular subtenants by setting additional tenant options, for example via tenant policy. For details, see the Default subscriptions screenshot below or the [Tenant API](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#tag/Tenant-API) in the {{< openapi >}}.
```
[the remark this answers is not in the document]
Comment (dianaolives) [reply content/users-guide/enterprise-tenant-bundle/managing-tenants.md:352]: The section that was tagged was the exact section where this was written and the text is actually making reference to a screenshot below. This was what the ticket said: "there is a link added for text Default subscriptions => so the text is in blue but doesn’t navigate anywhere, link should be removed".

File: content/users-guide/enterprise-tenant-bundle/support-user-access.md (hunk 3 above)
Comment (dianaolives) [reply content/users-guide/enterprise-tenant-bundle/support-user-access.md:50]: Ok

File: content/users-guide/enterprise-tenant-bundle/data-broker.md [hunk 5]
```diff
@@ -167,6 +167,10 @@ Click the menu icon at the right of a data connector entry and then click **Dupl
 
 Click the menu icon at the right of a data connector entry and then click **Delete** to stop data forwarding and delete the data connector.
 
+{{< c8y-admon-info >}}
+A data connector can be removed from the connector settings page.
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/data-broker.md:171]: connector settings page > This is not very precise. As far as I can see, it is the **Settings"" tab of a connector configuration.

What I do not get is what this "Remove" option does. Can you please explain @sagscmi ?

File: content/users-guide/enterprise-tenant-bundle/managing-tenants.md [hunk 6]
```diff
@@ -39,17 +39,20 @@ To be able to use the tenant functionality, your user needs to have the appropri
...
 * The name of the subtenant, for example, company name of your customer.
 * The ID and domain.
-* Optionally, a contact name and phone number.
+* Optionally, a contact name.
 * The date when the tenant was created.
 * The status of the tenant, either active (indicated by a green checkmark icon) or suspended (indicated by a red cross icon).
 
-In the {{< management-tenant >}}, you will also find information on the parent tenant, that is, the tenant that created the listed tenant.
+{{< c8y-admon-info >}}
+In the {{< management-tenant >}}, you will also find a column with information on the parent tenant. The parent tenant is the original tenant that created the subtenants that are listed in the table.
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/managing-tenants.md:53]:
```suggestion
In the {{< management-tenant >}}, you also find a column with information on the parent tenant. The parent tenant is the original tenant that created the subtenants that are listed in the table.
```

File: content/users-guide/enterprise-tenant-bundle/data-broker.md (hunk 5 above)
Comment (sagscmi) [reply content/users-guide/enterprise-tenant-bundle/data-broker.md:171]: It removes a previously created data broker connector.  From what I can see (may be looking at the wrong version) there is a "remove" option on the three-dots menu for the connector when you're on the page showing all your connectors, or if you have opened the properties for a specific connector there is a "Remove" button at the bottom of the page.  They both do the same thing.

Comment (BeateRixen) [reply content/users-guide/enterprise-tenant-bundle/data-broker.md:171]: > It removes a previously created data broker connector. From what I can see (may be looking at the wrong version) there is a "remove" option on the three-dots menu for the connector when you're on the page showing all your connectors, or if you have opened the properties for a specific connector there is a "Remove" button at the bottom of the page. They both do the same thing.

Bad wording then. In the context menu it is called "Delete". Really confusing.

Comment (sagscmi) [reply content/users-guide/enterprise-tenant-bundle/data-broker.md:171]: > > It removes a previously created data broker connector. From what I can see (may be looking at the wrong version) there is a "remove" option on the three-dots menu for the connector when you're on the page showing all your connectors, or if you have opened the properties for a specific connector there is a "Remove" button at the bottom of the page. They both do the same thing.
> 
> Bad wording then. In the context menu it is called "Delete". Really confusing.

TBH I never looked that closely... we can certainly log a defect to fix that.

[the remark this answers is not in the document]
Comment (dianaolives) [reply content/users-guide/enterprise-tenant-bundle/data-broker.md:171]: OK

File: content/users-guide/enterprise-tenant-bundle/managing-tenants.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (dianaolives) [reply content/users-guide/enterprise-tenant-bundle/managing-tenants.md]: Added link to Overriding default subscriptions section below

File: content/users-guide/enterprise-tenant-bundle/customization.md [hunk 7]
```diff
@@ -27,6 +27,11 @@ The following placeholders can be found in the **Configuration** tab:
 |{tenant-domain}|This is the location in which a tenant can be accessed. It is equal to "https://" + "&lt;&lt;tenantDomainName&gt;&gt;". For example, {tenant-domain} can be `https://myTenant.{{< domain-c8y >}}`. In case of an {{< enterprise-tenant >}}, the {tenantDomain} placeholders can have different values. An example tenant domain is `https://myTenant.myhost.com`.
 |{token}|An automatically generated system token for password reset purposes. When a user requests a password reset, a new random token will be generated. This token will be associated only with the particular user and will allow for a single password reset action. The standard way of using this placeholder is along with the {tenant-domain} property as "{tenant-domain}?token={token}".
 |{email}|This placeholder will be replaced with the email address of the recipient user as stored in the user settings. Some views in the UI recognize this parameter and prefill the respective field with this value, for example, during the process of password reset.
+|{username}|
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/customization.md:30]: @Airdroplet According to the ticket from Daria, these placeholders are missing. If so, could you please provide the descriptions?

**Comment by BeateRixen** [comment]
@Airdroplet This is the PR for the ticket https://cumulocity.atlassian.net/browse/MTM-50953. Please review and approve or add more people that might be relevant. We implemented most of Darias comments but cannot judge the correctness of some of them.

File: content/users-guide/enterprise-tenant-bundle/customization.md (hunk 7 above)
Comment (Airdroplet) [reply content/users-guide/enterprise-tenant-bundle/customization.md:30]: provided

File: content/users-guide/enterprise-tenant-bundle/managing-tenants.md [hunk 8]
```diff
@@ -315,7 +326,7 @@ For each tenant policy, the name, an optional description and the number of opti
 The tenant policy will be added to the tenant policies list.
 
 {{< c8y-admon-important >}}
-When defining the retention rules and options you can select a checkbox to allow subtenants to modify definitions of these rules or options. By default, this checkbox is not activated. Be aware that if you do not select this checkbox after creating the subtenant you must run an update from the {{< management-tenant >}} in order to edit those rules and options.
+**Check this -->** When defining the retention rules and options you can select a checkbox to allow subtenants to modify definitions of these rules or options. By default, this checkbox is not activated. Be aware that if you do not select this checkbox after creating the subtenant you must run an update from the {{< management-tenant >}} in order to edit those rules and options.
```
[the remark this answers is not in the document]
Comment (Airdroplet) [reply content/users-guide/enterprise-tenant-bundle/managing-tenants.md:329]: Not really, it is visible for both managmeent and ee within tenant policies edit view.
And subtenant doesn't have tenant policies by definition.

 Visible here:
image


But not here: 
image

**Comment by Airdroplet** [comment]
@BeateRixen I think we need to ask @carlosceia to provide some feedback on replacing some values for branding customization screenshot, or remove it completely: /images/users-guide/enterprise-tenant/et-branding-preview.png
et-branding-preview

used in content/users-guide/enterprise-tenant-bundle/customization.md:106

I felt sick during documentation review when saw that color palette XD

**Comment by Airdroplet** [comment]
I pushed some changes, everything seems ok. With one trivial question about whenever we should replace the screenshot for explaining branding preview feature.

File: content/users-guide/enterprise-tenant-bundle/customization.md [hunk 9]
```diff
@@ -27,6 +27,14 @@ The following placeholders can be found in the **Configuration** tab:
...
 |{token}|An automatically generated system token for password reset purposes. When a user requests a password reset, a new random token will be generated. This token will be associated only with the particular user and will allow for a single password reset action. The standard way of using this placeholder is along with the {tenant-domain} property as "{tenant-domain}?token={token}".
 |{email}|This placeholder will be replaced with the email address of the recipient user as stored in the user settings. Some views in the UI recognize this parameter and prefill the respective field with this value, for example, during the process of password reset.
+|{username}|This placeholder will be replaced with the value of username property specified within user configuration, see: [User options and settings](/users-guide/getting-started/#user-settings).
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/customization.md:30]:
```suggestion
|{username}|Will be replaced with the value of the username property specified in the user configuration, see [User options and settings](/users-guide/getting-started/#user-settings).
```

File: content/users-guide/enterprise-tenant-bundle/customization.md [hunk 10]
```diff
@@ -27,6 +27,14 @@ The following placeholders can be found in the **Configuration** tab:
...
+|{binaryId}|This placeholder will be replaced with respective binaryId for binary artifact for use in download link.
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/customization.md:31]:
```suggestion
|{binaryId}|Will be replaced with the respective `binaryId` for the binary artefact being used in the download link.
```

File: content/users-guide/enterprise-tenant-bundle/customization.md [hunk 11]
```diff
@@ -27,6 +27,14 @@ The following placeholders can be found in the **Configuration** tab:
...
+|{exportApi}|This placeholder will be replaced with respective API on which error occured.
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/customization.md:32]:
```suggestion
|{exportApi}|Will be replaced with the respective API in which the error occurred.
```

File: content/users-guide/enterprise-tenant-bundle/customization.md [hunk 12]
```diff
@@ -27,6 +27,14 @@ The following placeholders can be found in the **Configuration** tab:
...
+|{size}|This placeholder will be replaced with storage usage percentage value.
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/customization.md:33]:
```suggestion
|{size}|Will be replaced with the storage usage percentage value.
```

File: content/users-guide/enterprise-tenant-bundle/customization.md [hunk 13]
```diff
@@ -26,7 +26,15 @@ The following placeholders can be found in the **Configuration** tab:
...
-|{email}|This placeholder will be replaced with the email address of the recipient user as stored in the user settings. Some views in the UI recognize this parameter and prefill the respective field with this value, for example, during the process of password reset.
+|{email}|Will be replaced with the email address of the recipient user as stored in the user settings. Some views in the UI recognize this parameter and prefill the respective field with this value, for example, during the process of password reset.
+|{username}|Will be replaced with the value of the username property specified in the user configuration, see [User options and settings](/users-guide/getting-started/#user-settings).
+|{binaryId}|Will be replaced with the respective `binaryId` for the binary artefact to be used in the download link.
+|{exportApi}|Will be replaced with the respective API in which the error occurred.
+|{size}|Will be replaced with the storage usage percentage value.
+
+{{< c8y-admon-info >}}
+Note that above mentioned placeholders might not be applicable to certain templates. While preparing content, be adviced by contextual information provided within user interface.
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/customization.md:36]:
```suggestion
The above mentioned placeholders might not be applicable to certain templates. While preparing content, note the information provided in the UI.
```

**Review by Airdroplet — APPROVED** [review]

File: content/users-guide/enterprise-tenant-bundle/customization.md [hunk 14]
```diff
@@ -26,7 +26,15 @@ The following placeholders can be found in the **Configuration** tab:
...
 |{token}|An automatically generated system token for password reset purposes. When a user requests a password reset, a new random token will be generated. This token will be associated only with the particular user and will allow for a single password reset action. The standard way of using this placeholder is along with the {tenant-domain} property as "{tenant-domain}?token={token}".
-|{email}|This placeholder will be replaced with the email address of the recipient user as stored in the user settings. Some views in the UI recognize this parameter and prefill the respective field with this value, for example, during the process of password reset.
+|{email}|Will be replaced with the email address of the recipient user as stored in the user settings. Some views in the UI recognize this parameter and prefill the respective field with this value, for example, during the process of password reset.
```
Comment (BeateRixen) [inline content/users-guide/enterprise-tenant-bundle/customization.md:29]: @dianaolives @Airdroplet Not sure about the sorting. Should it better be alphabetically? Or is there an order in which these placeholders appear?

**Comment by BeateRixen** [comment]
> @BeateRixen I think we need to ask @carlosceia to provide some feedback on replacing some values for branding customization screenshot, or remove it completely: /images/users-guide/enterprise-tenant/et-branding-preview.png et-branding-preview
> 
> used in content/users-guide/enterprise-tenant-bundle/customization.md:106
> 
> I felt sick during documentation review when saw that color palette XD

@Airdroplet What is the issue with the screenshot? Only that you dislike the colors :-)? Or which values need to be changed? Perhaps @eniosultan can help?

**Review by jkot-softwareag — APPROVED** [review]

**Comment by Airdroplet** [comment]
> @Airdroplet What is the issue with the screenshot? Only that you dislike the colors :-)? Or which values need to be changed? Perhaps @eniosultan can help?

Mainly color palette, black font on green background, while selected item is highlighted with white color. I just thought that it might be better if we follow some basic design rules for the screenshots we present as example of configuration. It was too contrasting for my eyes :)

**Comment by BeateRixen** [comment]
> > @Airdroplet What is the issue with the screenshot? Only that you dislike the colors :-)? Or which values need to be changed? Perhaps @eniosultan can help?
> 
> Mainly color palette, black font on green background, while selected item is highlighted with white color. I just thought that it might be better if we follow some basic design rules in terms of selecting palette for the screenshots we present as example of configuration. It was too contrasting for my eyes :)

I agree. @eniosultan could you please provide a screenshot which better meets our design standards?

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**